Doc/using/cmdline.rst

```diff
@@ -1054,7 +1054,14 @@ the :mod:`site` module documentation.
 .. note::
 
    The effect of :mod:`site` on the module search path may be seen by running
-   Python with ``-S`` which will start Python without importing site.
+   Python with ``-S`` which will start Python without importing site. ``-s`` will
+   skip adding the :data:`user site-packages directory <site.USER_SITE>` to :data:`sys.path`.
+
+   The effect of :envvar:`PYTHON*` environment variables may be seen by running
+   Python with ``-E`` which will start Python ignoring these environment variables.
+
+   Finally, ``-I`` will start Python in isolated mode, simlar to ``-E`` and ``-s``
+   but with further restrictions.
```

Comment:
This whole note feels a bit backwards - useful tips for someone trying to diagnose an issue, but not really documentation.

Maybe frame it as "Certain command line options may further affect path calculations. See `-E`, `-I`, `-S` and `-s` for further details." (and make each option a link to its docs, further up on this page).

Reply:
Ok, done.